sample-apps/go-sample-app/collection/client.go

```diff
@@ -120,29 +124,3 @@ func setupTraceProvider(ctx context.Context) (*sdktrace.TracerProvider, error) {
 	)
 	return tp, nil
 }
-
-// setupMetricsController configures a metric exporter and a controller with a histogram tracking latency.
-func setupMetricsController(ctx context.Context) (*controller.Controller, error) {
-	metricClient := otlpmetricgrpc.NewClient(
-		// INSECURE !! NOT TO BE USED FOR ANYTHING IN PRODUCTION
-		otlpmetricgrpc.WithInsecure(),
-		otlpmetricgrpc.WithEndpoint(grpcEndpoint))
-	metricExp, _ := otlpmetric.New(ctx, metricClient)
-
-	controller := controller.New(
-		processor.NewFactory(
-			selector.NewWithHistogramDistribution(
-				histogram.WithExplicitBoundaries([]float64{100, 300, 500}), // Tracking latency
-			),
```

Comment:
I don't see anything setting bucket boundaries anymore.  What happened to them?

Reply:
The other sample apps did not specify bucket boundaries so I did not think it was needed here.  Is this something we should add?  If so, how should I set this for the sample apps.

Reply:
Added the bucket boundaries to the exporter in latest commit.